async-inference/HuggingFace-Async-Inference-Walkthrough.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>Use doc page title as link text, like:</p><p> </p><p><code>[Registry Paths and Example Code](https://github.com/awsdocs/amazon-sagemaker-developer-guide/blob/master/doc_source/ecr-eu-west-1.md#huggingface-eu-west-1.title)</code></p><p> </p><p>This link is only for eu-west-1 - where are other regions documented?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2910/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
updated the link to point to the parent folder and advised customer to choose the appropriate ecr container image from the region of their interest